src/core/dynamic_batch_scheduler.cc

```diff
@@ -197,9 +197,6 @@ DynamicBatchScheduler::SchedulerThread(
                  << model_->Name() << " at default nice...";
 #endif
 
-  // TODO: Move Initialize and WarmUp to triton_model and
```

Comment:
@deadeyegoodwin  I think we would need to leave the `Initialize` and `WarmUp` call as is instead of earlier proposed idea of moving into construction logic. With @GuanLuo changes we would want to call these functions on the thread which is going to enter the backend exec. What do you think?

Reply:
ok